poms/bom-test/pom.xml

```diff
@@ -34,8 +36,10 @@
 
     <properties>
         <!-- This property is kept in sync with project.version by the release plugin -->
-        <!-- Do not change to project.version because otherwise the end user apps having our BOM as parent (rather than importing it) will stop working -->
+        <!-- Do not change to project.version because otherwise the end user apps having our BOM as
+        parent (rather than importing it) will stop working -->
         <camel-quarkus.version>3.0.0-SNAPSHOT</camel-quarkus.version>
+        <unboundid-ldapsdk.version>6.0.8</unboundid-ldapsdk.version>
```

Comment:
Please can you move the `unboundid-ldapsdk.version` property to the root `pom.xml`.

There is a section for test related items:

https://github.com/apache/camel-quarkus/blob/main/pom.xml#L162

Sorry - I should have mentioned that in my previous comment.

Reply:
Fixed